pennylane/qinfo/classical_fisher.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Differentiable classical fisher information"""
+# pylint: disable=import-outside-toplevel
+import functools
+import pennylane as qml
+
+from pennylane.transforms import batch_transform
+
+
+def _torch_jac(circ):
```

Comment:
You could go in the math module in `single_dispatch` and create a `qml.math.jac`, I did it for entropy you could check this PR-4. What do you think about that @josh146 ?

Reply:
While that would be nice to have, I would say this is out of scope for this PR :)